azkaban-common/src/test/java/azkaban/executor/SleepJavaJob.java

```diff
@@ -88,7 +88,7 @@ public void run() throws Exception {
     System.out.println("Sec " + sec);
     synchronized (this) {
       try {
-        this.wait(sec * 1000);
+        this.wait(sec * 1000 + 1);
```

Comment:
wait(1ms) when sec=0. wait(0ms) waits forever so that's why this is better.

Reply:
Thanks for the explanation.

Is it to be expected that sometimes sec will be 0?

If so, would it be better to add a comment to the code since it is not obvious?

Reply:
> Is it to be expected that sometimes sec will be 0?

Yes

> If so, would it be better to add a comment to the code since it is not obvious?

In that case, what would you think about this instead:

```java
if (sec > 0) {
   this.wait(sec * 1000);
}
```
?

I wanted to avoid such verbosity, but if the +1 way is not clear enough, then I'd rather handle it with self-documenting code like this instead of explaining in a comment.

Reply:
The sec seems to control how many seconds to wait before failing the job. Am I right? 

Could you point me to a test that sets the wait time to 0? I can search for it. But since you are more familiar with this code, it's easier for me to ask. 

If 0 value is expected, then wait doesn't make logical sense, even with wait(1). Yes, your suggested alternative looks better to me. 

>  I'd rather handle it with self-documenting code like this instead of explaining in a comment.

In general, yes I agree with you.

Reply:
Ping.

Reply:
@juhoautio I am happy to continue to review and accept this PR if you can respond to the comments.